cocos/network/SocketIO.h

```diff
@@ -255,12 +258,32 @@ class CC_DLL SIOClient
      * @param s message.
      */
     void send(const std::string& s);
+    void send(const std::list<std::string>& s);
```

Comment:
`void send(const std::list<std::string>& s)` -> `void send(const std::list<std::vector>& s)`

Reply:
I think vector is better here since it is more efficient for loop